enterprise/frontend/src/metabase-enterprise/whitelabel/components/ImageUploadInfoDot.tsx

```diff
@@ -0,0 +1,50 @@
+import { useTheme } from "@emotion/react";
+import { t } from "ttag";
+
+import { Icon, Stack, Text, HoverCard } from "metabase/ui";
+
+type IllustrationType = "background" | "icon";
+
+interface CustomFileUploadInfoDot {
+  type: IllustrationType;
+}
+
+const DESCRIPTIONS_WIDTHS: Record<IllustrationType, number> = {
+  background: 400,
+  icon: 250,
+};
+
+const DESCRIPTIONS: Record<IllustrationType, string[]> = {
```

Comment:
Not sure if this is too abstracted. I could inline it, but it makes it a little bit more messy if I inline put all the JSX elements here.